web/src/components/organisms/Project/Schema/index.tsx

```diff
@@ -1,23 +1,15 @@
 import styled from "@emotion/styled";
-import React, { useState } from "react";
+import React from "react";
```

Comment:
You should be able to remove this completely. I believe CMS is (or should be) setup with the Vite react plugin, which will inject the import React in at build if its not there.

Reply:
https://github.com/reearth/reearth-cms/blob/6e45c9fb406680bf370be1b073859163593f42ba/web/src/components/organisms/Dashboard/index.tsx#L2